google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/stub/mutaterows/MutateRowsBatchingDescriptorTest.java

```diff
@@ -77,29 +78,35 @@ public void requestBuilderTest() {
 
   @Test
   public void splitResponseTest() {
-    List<SettableApiFuture<Void>> batchResponse =
-        ImmutableList.of(SettableApiFuture.<Void>create(), SettableApiFuture.<Void>create());
-    assertThat(batchResponse.get(0).isDone()).isFalse();
-    assertThat(batchResponse.get(1).isDone()).isFalse();
+    BatchEntry<RowMutationEntry, Void> resultFuture1 = BatchEntry.create(null, SettableApiFuture.<Void>create());
+    BatchEntry<RowMutationEntry, Void> resultFuture2 = BatchEntry.create(null, SettableApiFuture.<Void>create());
+
```

Comment:
The mutaterows descriptor shouldn't accept nulls, the check is missing, but lets keep the test correct:

```java
List<BatchEntry<RowMutationEntry, Void>> entries =
        ImmutableList.of(
            BatchEntry.create(
                RowMutationEntry.create("key1").deleteRow(), SettableApiFuture.<Void>create()),
            BatchEntry.create(
                RowMutationEntry.create("key2").deleteRow(), SettableApiFuture.<Void>create()));
```